Documentation/spelling_wordlist.txt

```diff
@@ -810,6 +812,9 @@ tracepoint
 tracepoints
 tracestate
 transpiling
+Triager
+Triaging
+triaging
```

Comment:
```suggestion
triaging
```

Nit: As `triaging` contains no capital letter, it's considered case-insensitive by the spell checker and should be enough to make `Triaging` considered as a valid word. So `Triaging` is not required here.